lib/peep.rb

```diff
@@ -0,0 +1,27 @@
+require 'pg'
+require_relative 'database_connection'
+
+class Peep
+
+  
+  def self.all
+
+    if ENV['ENVIRONMENT'] == 'test'
+      DatabaseConnection.setup('chitter_test')
+    else
+      DatabaseConnection.setup('chitter')
+    end
+
+    raw_peeps = DatabaseConnection.query('SELECT * FROM peeps')
+
+    peeps = raw_peeps.map do |item|
+      { timestamp: item['timestamp'][0, 19], message: item['peep'] }
+    end
+
+    peeps.reverse
```

Comment:
good solution! but a different approach; sql lets you make ascending/descending order, so you can select it like this: 'SELECT * FROM peeps ORDER BY timestamp DESC;' - so no need the extra line to reverse it :)

Reply:
Good to know! I did not much look into SQL so that Ruby way was the only way I knew how to do it